modules/nf-core/busco/main.nf

```diff
@@ -14,11 +14,16 @@ process BUSCO {
     path config_file                      // Optional:    busco configuration file
 
     output:
-    tuple val(meta), path("*-busco.batch_summary.txt"), emit: batch_summary
-    tuple val(meta), path("short_summary.*.txt")      , emit: short_summaries_txt, optional: true
-    tuple val(meta), path("short_summary.*.json")     , emit: short_summaries_json, optional: true
-    tuple val(meta), path("*-busco")                  , emit: busco_dir
-    path "versions.yml"                               , emit: versions
+    tuple val(meta), path("*-busco.batch_summary.txt")                , emit: batch_summary
+    tuple val(meta), path("short_summary.*.txt")                      , emit: short_summaries_txt, optional: true
+    tuple val(meta), path("short_summary.*.json")                     , emit: short_summaries_json, optional: true
+    tuple val(meta), path("*-busco/*/run_*/full_table.tsv")           , emit: full_table, optional: true
+    tuple val(meta), path("*-busco/*/run_*/missing_busco_list.tsv")   , emit: missing_busco_list, optional: true
+    tuple val(meta), path("*-busco/*/run_*/single_copy_proteins.faa") , emit: single_copy_proteins, optional: true
+    tuple val(meta), path("*-busco/*/run_*/busco_sequences")          , emit: seq_dir
```

Comment:
It's new but non-optional, is that righ?

Reply:
Correct. It's present in all the runs I've seen so far, and all the combinations of options tested in nf-core.